src/routes/sections/HeroSection.svelte

```diff
@@ -16,6 +16,7 @@
     display: flex;
     flex-direction: column;
     gap: 2rem;
+
     @include mq("medium") {
       max-width: 60%;
```

Comment:
Two comments here:

1) Same as the Footer container width. Try to use a constrained amount of container widths to make the page look tidier.

2) If what you were trying to make is texts don't become too stretched, `%` is not the correct unit here.
Think of a super wide screen, this container will expand a lot because the % goes tied to the width of the parent element, and this 60% will be suuuuuper wide.

To constrain text widths the perfect unit is `ch`: https://daily-dev-tips.com/posts/a-look-at-the-ch-css-unit/

Conclusion: 

All in all, I believe using a `.container-small` everything gets solved.
If you see the styling of that container it uses a `min()` (https://developer.mozilla.org/en-US/docs/Web/CSS/min) to fix its width. 
By using min, we prevent the `%` of making suuuper wide containers in wide screens.